radixdlt-java/src/main/java/com/radixdlt/client/application/RadixApplicationAPI.java

```diff
@@ -124,6 +124,14 @@ private Result connect() {
 		public Completable toCompletable() {
 			return completable;
 		}
+
+		/**
+		 * Block until the execution of the action is stored on the node ledger.
+		 * Throws an exception if there are any issues.
+		 */
+		public void blockUntilComplete() {
```

Comment:
This does not appear to be used yet, is this no longer necessary or just an addition?

Reply:
used in RegressionTest